_posts/salads/2016-07-23-greenbean-tomato-shallot.md

```diff
@@ -0,0 +1,19 @@
+Adapted from [Smitten Kitchen](https://smittenkitchen.com/2008/05/green-bean-and-cherry-tomato-salad/) / Chez Panisse Vegetables
+
+When I lived in San Francisco the first time (2012-2013), I found a copy of Chez Panisse Vegetables and kept it as my toilet reading (TMI for a food blog?) for a solid few months. Facing a never ending CSA box of vegetables I had never ever cooked in my life, it was a godsend. Since, I've relapsed into more Chinese cooking at home, but I thank both Eatwell Farms (best CSA / community farm ever) and that book for coaching me through eating seasonally.
+
+ I never made this salad then, though. I made this for the first time in NYC with Greenmarket (don't ask me why they call it that in NYC and not a farmer's market, I have no idea either) green beans and cherry tomatoes. 
+
+- 1 pound green beans (or more)
+- 1 pound cherry tomatoes (or more, 'cause you're going to eat a whole pint on the way home from the market)
+- at least 4 shallots 
+
+A day ahead (optionally), make some _shallot oil_ with the shallots. Slice shallots evenly and thinly (don't dice), and heat up a cup of neutral high heat point oil (I like canola, but grapeseed is good too), and fry until the shallots crisp up and resemble what you'd put on top of green bean casserole. Carefully and quickly once you reach the brown point, remove the shallots to a paper towel'd plate and salt generously. Let cool so that they get crisp and delicious. 
```

Comment:
Hmmmm ... I generally find it easier to easier to reference these recipes when these segments are organized/formatted in bullet points, but I think doing so would destroy the charm of the paragraph. Something like [this](https://github.com/mookerzhou/cooking_journal/blob/master/_posts/pastry/2018-01-03-lemon-bars.md) is what I had in mind.